tests/general/LineCountBearTest.py

```diff
@@ -25,3 +25,17 @@ def test_run(self):
             filename='default')
         self.check_validity(self.uut, ['1 line'])
         self.check_validity(self.uut, [])  # Empty file
+
+    def test_exclude_blank_lines(self):
+        self.section.append(Setting('exclude_blank_lines', True))
+        self.section.append(Setting('max_lines_per_file', 2))
+        self.check_results(
+            self.uut, ['line 1', ' ', 'line 2',
+                       'line 3', '\n', '\t', ' line 4',
```

Comment:
probably also a good idea to have one empty line with multiple mixed whitespace characters in it; currently they are all single character lines.